<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>elly_qtmaindialog</class>
 <widget class="QDialog" name="elly_qtmaindialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>589</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_left" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="parameters">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <row>
         <property name="text">
          <string>clado_is</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>clado_main</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>ana</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>ext_is</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>ext_main</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>mig_to_is</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>carryingcap_is</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>carryingcap_main</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>rng_seed</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>init_n_mainland</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>crown_age</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>?</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>?</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>?</string>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>?</string>
         </property>
        </item>
        <item row="4" column="0">
         <property name="text">
          <string>?</string>
         </property>
        </item>
        <item row="5" column="0">
         <property name="text">
          <string>?</string>
         </property>
        </item>
        <item row="6" column="0">
         <property name="text">
          <string>?</string>
         </property>
        </item>
        <item row="7" column="0">
         <property name="text">
          <string>?</string>
         </property>
        </item>
        <item row="8" column="0">
         <property name="text">
          <string>?</string>
         </property>
        </item>
        <item row="9" column="0">
         <property name="text">
          <string>?</string>
         </property>
        </item>
        <item row="10" column="0">
         <property name="text">
          <string>?</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="start">
        <property name="text">
         <string>&amp;Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_sim_runtime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Simulation runtime: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_right" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_rates">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Rates</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
